packages/app/src/components/ConfirmModal/index.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/jsx-wrap-multilines */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal } from 'react-rainbow-components';
+import { Header, Question, Container, IconContainer, ButtonsContainer, Button } from './styled';
+import { defaultCancelButtonLabel, defaultOkButtonLabel, defaultTitle } from './labels';
+
+const ConfirmModal = (props) => {
+    const {
+        header,
+        icon,
+        question,
+        variant,
+        isOpen,
+        onCancel,
+        onConfirm,
+        cancelButtonLabel,
+        okButtonLabel,
+    } = props;
+    const title = header || defaultTitle;
+    const cancelLabel = cancelButtonLabel || defaultCancelButtonLabel;
+    const okLabel = okButtonLabel || defaultOkButtonLabel;
```

Comment:
We should not use keys internationalization here because the user controls content to show.

Reply:
i left the the default labels in defaultProps...